MapboxVision/src/main/java/com/mapbox/vision/VisionManager.kt

```diff
@@ -218,32 +230,11 @@ object VisionManager : BaseVisionManager {
             return
         }
 
-        sessionManager = SessionManager.RotatedBuffersImpl(
-            application,
-            nativeVisionManager,
-            delegate.rootTelemetryDir,
-            videoRecorder,
-            telemetryImageSaver,
-            VideoProcessor.Impl(),
-            videoProcessorListener
-        )
-
         delegate.start(
             visionEventsListener
         ) { country ->
             handlerMain.post {
-                currentCountry = country
-
-                if (!isRecording) {
-                    when (country) {
-                        Country.China -> {
-                            sessionManager.stop()
-                        }
-                        Country.Unknown, Country.UK, Country.USA, Country.Other -> {
-                            sessionManager.start()
-                        }
-                    }
-                }
+                sessionManager.setCountry(country)
```

Comment:
Is `handlerMain.post(sessionManager::setCountry)` possible here?

Reply:
not possible. leave current version